ui/widgets/nuxeo-html-editor.js

```diff
@@ -39,8 +40,82 @@ import './quill/quill-snow.js';
           div#editor > * {
             margin-top: 0;
           }
+
+          iron-icon {
+            height: 18px;
+            color: #444;
+          }
+
+          nuxeo-document-picker {
+            --nuxeo-document-picker-dialog-max-height: 84%;
```

Comment:
Magic numbers? 😂

Reply:
These are [part of the spec](https://jira.nuxeo.com/browse/WEBUI-167?focusedCommentId=517994&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-517994), so I assume there's some reasoning behind them.

[This 520px is definitely "magical"](https://github.com/nuxeo/nuxeo-elements/commit/83793d405cf49994baad7e883bbdabc25fdb947c#diff-4663ded333a21db2537f4eba0f4e208da986f53eb5f4157fae49af9161b32900R69), which probably means that we should also expose a `var` for it. If you're changing the `max-height` of the dialog you're probably going to need to adjust the height of the results view as well. WDYT @Gabez0r?

Reply:
I'd be ok to add a new variable, since it's just a matter of time before someone asks to make it larger. :P